index.php

```diff
@@ -1,18 +1,22 @@
 <?php
 
-use TheRealMVP\Importers\DBImport;
-use TheRealMVP\Importers\GetAPI;
-use TheRealMVP\Importers\PDO;
-use TheRealMVP\DisplayHelpers\DisplayData;
+session_start();
+$_SESSION['sport'] = $_GET['sport'] ?? $_SESSION['sport'];
+$_SESSION['country'] = $_GET['country'] ?? $_SESSION['country'];
+
 use TheRealMVP\Hydrators\TeamHydrator;
+use TheRealMVP\Hydrators\SportHydrator;
+use TheRealMVP\Hydrators\CountryHydrator;
+use TheRealMVP\DisplayHelpers\DisplayFilter;
+use TheRealMVP\DisplayHelpers\DisplayData;
 
 require_once './vendor/autoload.php';
+require_once 'importer.php';
```

Comment:
This shouldn't be, it will result in the db being refreshed on every page load

Reply:
Removed